torch/distributed/pipelining/_debug.py

```diff
@@ -1,9 +1,10 @@
-# mypy: allow-untyped-defs
 # Copyright (c) Meta Platforms, Inc. and affiliates
+
 import torch
+from torch.fx.node import Argument
 
 
-def friendly_debug_info(v):
+def friendly_debug_info(v: object) -> Argument:
```

Comment:
Umm, how is a `str` an Argument? Doesn't seem to match the typing info for it unless I am missing something

Reply:
It's a `BaseArgumentTypes` in torch.fx.node

```
BaseArgumentTypes = Union[
    str,
    int,
    float,
    bool,
    complex,
    torch.dtype,
    torch.Tensor,
    torch.device,
    torch.memory_format,
    torch.layout,
    torch._ops.OpOverload,
    torch.SymInt,
    torch.SymBool,
    torch.SymFloat,
]
base_types = BaseArgumentTypes.__args__  # type: ignore[attr-defined]

Target = Union[Callable[..., Any], str]

Argument = Optional[
    Union[
        Tuple["Argument", ...],
        Sequence["Argument"],
        Mapping[str, "Argument"],
        slice,  # Slice[Argument, Argument, Argument], but slice is not a templated type in typing
        range,
        "Node",
        BaseArgumentTypes,
    ]
]
```